### DISCUSSION TITLE
Looks like state is not in sync

### DISCUSSION THREAD

**Discussion by dev-cyprium** [Akryum/v-tooltip#360] [Q&A]
Hello, great plugin by the way!

However, I've stumbled upon just one weird issue with it:

I have a computed property `isOpen` which does the following code:
```js
/* This is my Tutorial component that uses the v-popover */
{
data() {
    return {
      open: true
    }
  }
computed: {
    isOpen() {
      if (this.tutorialDescription === null) {
        return false
      }

      return this.open
    },
    tutorialDescription() {
      const tut = this.$store.getters.tutorialDescription(this.feature)

      if (tut === 'LOADING') {
        return 'Loading...'
      }

      if (tut === null) {
        return null
      }

      return tut;
    }
  }
}
```

And here is the template:
```
<template>
  <div>
    <v-popover placement="top" trigger="manual" :open="isOpen" :auto-hide="false" offset="8">
      <slot :dissmiss="this.dissmiss.bind(this)" />
      <template v-slot:popover>
        <p>{{tutorialDescription}}</p>
        <button @click="dissmiss()" type="button" class="button is-success is-small">Got it</button>
      </template>
    </v-popover>
  </div>
</template>
```
So, my computed property returns *false*, but the popover still is *being rendered*. Which means it's not in sync properly with the `prop`.

I assume this is the case because I refactored the code to use a `v-if` instead and a `v-else` on the same `isOpen` computed property, and wrapped the `v-popover` component and in that case it works flawlessly. However, I consider this a hack, and I'd like for the library to just work on it's own.

I also used the Vue dev tools to look into the state itself of the popper, and in fact, even though my prop value is *false*, the state of the component shows that it's internal state is open.

Maybe I'm doing something wrong? 
Regardless, thanks for the help, and keep up the good work on this plugin!

**Comment by lifenautjoe** [comment]
I have the same issue...
